### PR TITLE
John conroy/fix dev search

### PR DESCRIPTION
**Pull request by john-conroy**
Fixes dev search page following the changes to search.

### CODE REVIEW

File: context/app/static/js/pages/search/DevSearch.jsx [hunk 1]
```diff
@@ -22,12 +23,15 @@ function DevSearch() {
...
-      field('mapper_metadata.size', 'Doc Size'),
-    ],
+    resultFields: {
+      table: [
+        field('entity_type', 'type'),
+        field('display_doi', 'ID'),
+        field('mapped_last_modified_timestamp', 'Last Modified'),
+        field('mapper_metadata.size', 'Doc Size'),
+      ],
+      tile: [],
```
Comment (mccalluc) [inline context/app/static/js/pages/search/DevSearch.jsx:33]: Ditto from the earlier PR, wonder if we can just keep this flatter and request a superset.

File: context/app/static/js/stores/useSearchViewStore.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import create from 'zustand';
 
 const useSearchViewStore = create((set) => ({
-  searchView: new URLSearchParams(window.location.search).get('view') || 'table',
+  searchView:
+    (window.location.pathname.startsWith('/search') && new URLSearchParams(window.location.search).get('view')) ||
+    'table',
```
Comment (mccalluc) [inline context/app/static/js/stores/useSearchViewStore.js:6]: I'm not sure this is needed: `new URLSearchParams(window.location.search).get('view') || 'table'` will evaluate to `'table'` on the /dev-search.

**Review by mccalluc — CHANGES_REQUESTED** [review]
If it's an option, I think I would prefer a dev-search that had a tile view with some missing data to this. I worry about the little configuration details that are repeated in Results.jsx and DevResults.jsx, and I want to avoid creating new code to support dev-search. Maybe talk about this?

File: context/app/static/js/stores/useSearchViewStore.js (hunk 2 above)
Comment (john-conroy) [reply context/app/static/js/stores/useSearchViewStore.js:6]: Unless a user enters `&view=tile` in the url. Which we might not care about?

File: context/app/static/js/pages/search/DevSearch.jsx (hunk 1 above)
Comment (john-conroy) [reply context/app/static/js/pages/search/DevSearch.jsx:33]: We agreed this is fine for now, more in #1303.

File: context/app/static/js/stores/useSearchViewStore.js (hunk 2 above)
Comment (john-conroy) [reply context/app/static/js/stores/useSearchViewStore.js:6]: Per our Zoom conversation, we shouldn't worry about users modifying the url and things breaking. Prefer to keep code simpler.

**Pull request merged**